Футер (нужно проверить свг)

ArtiTemkin opened the pull request.

ArtiTemkin committed:
Футер (нужно проверить свг)

internet-user-0 commented on src/components/footer/footer.html:
```diff
@@ -1,10 +1,11 @@
-    <div class="container">
-        <p class="footer_text">&copy 2020 | All Rights Reserved | Developed with 
-            <svg>
+    <div class="container footer_container">
```
`footer__text` на всіх класах добав `_` вибиває помилку 
<img width="303" alt="image" src="https://user-images.githubusercontent.com/97742124/216829307-3c0f2733-b3e0-4c11-96be-8bdba7853232.png">

internet-user-0 commented on src/components/footer/footer.html:
```diff
@@ -1,10 +1,11 @@
+        <p class="footer_text">&copy 2020 | All Rights Reserved |
+            Developed with 
+            <svg width="14px" height="12.75px" class="footer_text-svg">
                 <use href="../../symbol-defs.svg#heart"></use>
```
`<use href="./symbol-defs.svg#heart"></use>` такий шлях

internet-user-0 reviewed (changes requested):
ще виправити треба

ArtiTemkin committed:
футер готов

internet-user-0 committed:
add/footer

добавив футер, для пред-ревю, потім Артем може його видалити, і зробити по своєму

internet-user-0 reviewed (approved):
+

internet-user-0 committed:
Merge branch 'main' into footer_styles
